### ISSUE TITLE
Missing pfps

### ISSUE THREAD

**Issue by CJMcKellar** [enchanted-sword/dashboard-unfucker#194]
**Describe the bug**
The pfps of the people I follow have disappeared from the side of the posts, so I can't tell who's reblogging what easily

**To Reproduce**
1. Go to your dashboard

**Screenshots**
![missing pfp](https://github.com/enchanted-sword/dashboard-unfucker/assets/135767462/7d9130c1-dbc3-46a3-b160-7d509357cb28)

**Version (please complete the following information):**
 - Firefox 120.0.1
 - Version 5.5.8

**Additional context**

**Comment by mount-weary** [comment]
having same issue on latest version of firefox 121.0 -- have cleared cache, restarted browser, reinstalled script, etc...
(pfps appear briefly as page is loading, then disappear once fully loaded)
*NOTE: the option to "re-add icons to reblogs" no longer appears under DU's advanced settings menu

**Comment by Psych-is-the-name** [comment]
having this same issue with Firefox 120.0.1

**Comment by kcdragons** [comment]
Also having this issue - Firefox 121.0 (updated today and problem started today)

**Comment by lassiepeahen** [comment]
im having the same issue—closed and reopened firefox today (12/20/23) and the application was still mostly working but the pfps were gone

**Comment by theladykit** [comment]
Having the same issue as of 12/20/23, Firefox 120.0.1. Installed most recent version of extension and restarted and nothing changed; the option to disable pfp scrolling is no longer under settings, either. Everything else seems to be working properly.

**Comment by BarryAllen2024** [comment]
also happening to me both on firefox 120.0.1 and 120.0

**Comment by CodeScop** [comment]
Same issue on my Firefox - here's what my Console looks like after I've tried to load:

```js
Autoplay is only allowed when approved by the user, the site is activated by the user, or media is muted. 3 [www.tumblr.com](https://www.tumblr.com/)
Content-Security-Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). 2 [utils.js:42:9](moz-extension://fe25e9ea-8024-a047-b88d-59907642b148/js/contentscripts/utils.js)
Object { acceptWebpUploads: true, activityHighlightUnread: true, activityRedesignM3: true, adFreeBrowsingGifts: true, adFreeCtaBanner: true, adFreeRedesign: true, adFreeSubscriptionStat: true, adsRefreshEnabled: true, adultContentAppealTextbox: true, androidHttp2Support: true, … }
line 3 > injectedScript:1716:15
Array(18) [ {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, {…}, … ]
line 3 > injectedScript:1717:15
unfucking dashboard...[ line 3 > injectedScript:1156:19](https://www.tumblr.com/%20line%203%20%3E%20injectedScript)
dashboard fixed![ line 3 > injectedScript:1713:17](https://www.tumblr.com/%20line%203%20%3E%20injectedScript)
init_extension: {"standard":true,"form_frame":false,"ask_frame":false,"blog_frame":false,"xkit":false} [xkit.js:33:12](moz-extension://710cb85a-7d5c-5340-8eaa-64258741985f/xkit.js)
Trying to run xkit_main. [xkit.js:110:14](moz-extension://710cb85a-7d5c-5340-8eaa-64258741985f/xkit.js)
Welcome from XKit Main 2.1.2 [xkit/xkit_main.js:45:12](chrome://devtools/content/webconsole/xkit/xkit_main.js)
Updates: difference = 1466140 [xkit/xkit_updates.js:34:12](chrome://devtools/content/webconsole/xkit/xkit_updates.js)
Skipping update checking. [xkit/xkit_updates.js:49:13](chrome://devtools/content/webconsole/xkit/xkit_updates.js)
Enabled extensions: xkit_main, xkit_patches, xkit_preferences, xkit_updates, xinbox, one_click_reply, tweaks, anti_capitalism, editable_reblogs, blacklist, notificationblock, old_blue, norecommended [xkit/xkit_main.js:126:12](chrome://devtools/content/webconsole/xkit/xkit_main.js)
Disabled extensions: go_to_dash (not in frame) [xkit/xkit_main.js:127:12](chrome://devtools/content/webconsole/xkit/xkit_main.js)
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://assets.tumblr.com/pop/js/modern/sentry-c4cf64c2.js?1703127654990. (Reason: CORS request did not succeed). Status code: (null).

<script> source URI is not allowed in this document: “https://assets.tumblr.com/pop/js/modern/sentry-c4cf64c2.js?1703127654990”. [www.tumblr.com:1:1](https://www.tumblr.com/)
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://assets.tumblr.com/pop/js/modern/sentry-c4cf64c2.js?1703127656202. (Reason: CORS request did not succeed). Status code: (null).

<script> source URI is not allowed in this document: “https://assets.tumblr.com/pop/js/modern/sentry-c4cf64c2.js?1703127656202”. [www.tumblr.com:1:1](https://www.tumblr.com/)
ChunkLoadError: Loading chunk 2897 failed after 2 retries.
(js/modern/sentry-c4cf64c2.js)
    j https://assets.tumblr.com/pop/js/modern/runtime-45041b4b.js:1
    e https://assets.tumblr.com/pop/js/modern/runtime-45041b4b.js:1
    e https://assets.tumblr.com/pop/js/modern/runtime-45041b4b.js:1
    e https://assets.tumblr.com/pop/js/modern/runtime-45041b4b.js:1
    e https://assets.tumblr.com/pop/js/modern/runtime-45041b4b.js:1
[main-9e7bca5a.js:128:3372](https://assets.tumblr.com/pop/js/modern/main-9e7bca5a.js)

​
```

**Comment by enchanted-sword** [comment]
They weren't showing up because Tumblr removed them entirely, but it's fixed in v5.6.0.
![image](https://github.com/enchanted-sword/dashboard-unfucker/assets/81981111/1ab4a1e6-7886-44b8-b5a3-b1f772580dbc)
